app/controllers/admin/users_controller.rb

```diff
@@ -4,7 +4,8 @@ def index
                                           params[:filterrific],
                                           select_options: {
                                             filter_volunteer_type: volunteer_type_options
-                                          })
+                                          },
+                                          persistence_id: false)
```

Comment:
Fixes the issue where the select filter on the Admin/User view would persist the value even if you navigated to another view then back again.

See here: https://github.com/jhund/filterrific/issues/86